--- cmake/os/Windows.cmake
+++ cmake/os/Windows.cmake
@@ -48,10 +48,10 @@
 GET_FILENAME_COMPONENT(_SCRIPT_DIR ${CMAKE_CURRENT_LIST_FILE} PATH)
 INCLUDE(${_SCRIPT_DIR}/WindowsCache.cmake)
 
-# We require at least Visual Studio 2017 (aka 15.8) which has version nr 1910.
-IF(NOT FORCE_UNSUPPORTED_COMPILER AND MSVC_VERSION LESS 1915)
+# We require at least Visual Studio 2015 (aka 14.0) which has version nr 1900.
+IF(NOT FORCE_UNSUPPORTED_COMPILER AND MSVC_VERSION LESS 1900)
   MESSAGE(FATAL_ERROR
-    "Visual Studio 2017 update 15.8 or newer is required!")
+    "Visual Studio 2015 or newer is required!")
 ENDIF()
 
 # OS display name (version_compile_os etc).
