--- include/typelib.h
+++ include/typelib.h
@@ -31,11 +31,11 @@
 
 struct MEM_ROOT;
 
-struct TYPELIB {             /* Different types saved here */
-  size_t count{0};           /* How many types */
-  const char *name{nullptr}; /* Name of typelib */
-  const char **type_names{nullptr};
-  unsigned int *type_lengths{nullptr};
+struct TYPELIB {    /* Different types saved here */
+  size_t count;     /* How many types */
+  const char *name; /* Name of typelib */
+  const char **type_names;
+  unsigned int *type_lengths;
 };
 
 extern my_ulonglong find_typeset(const char *x, TYPELIB *typelib,
--- include/mysql/components/services/psi_stage_bits.h
+++ include/mysql/components/services/psi_stage_bits.h
@@ -70,16 +70,16 @@
 */
 struct PSI_stage_info_v1 {
   /** The registered stage key. */
-  PSI_stage_key m_key{0};
+  PSI_stage_key m_key;
   /** The name of the stage instrument to register. */
-  const char *m_name{nullptr};
+  const char *m_name;
   /**
     The flags of the stage instrument to register.
     @sa PSI_FLAG_PROGRESS
   */
-  unsigned int m_flags{0};
+  unsigned int m_flags;
   /** Documentation. */
-  const char *m_documentation{nullptr};
+  const char *m_documentation;
 };
 typedef struct PSI_stage_info_v1 PSI_stage_info_v1;
 
